exercises/zipper/zipper.spec.js

```diff
@@ -0,0 +1,78 @@
+var Zipper = require('./zipper');
+
+function bt(value, left, right) {
+  return {
+    value: value,
+    left: left,
+    right: right
+  };
+}
+
+function leaf(value) {
+  return bt(value, null, null);
+}
+
+var t1 = bt(1, bt(2, null, leaf(3)), leaf(4));
+var t2 = bt(1, bt(5, null, leaf(3)), leaf(4));
+var t3 = bt(1, bt(2, leaf(5), leaf(3)), leaf(4));
+var t4 = bt(1, leaf(2), leaf(4));
+var t5 = bt(1, bt(2, null, leaf(3)), bt(6, leaf(7), leaf(8)));
+var t6 = bt(1, bt(2, null, leaf(5)), leaf(4));
+
```

Comment:
These variables seem to be the concern of the `describe` block, and (if I understand your intention) would not be used in a different `describe` block, so shouldn't they be declared inside the block?

The functions seem to be a different matter: they seem to be the sort of thing you might use repeatedly across `describe` blocks, so they seem to be in the right place.